pkgs/applications/networking/mailreaders/alot/default.nix

```diff
@@ -23,16 +25,18 @@ with python311.pkgs; buildPythonApplication rec {
   src = fetchFromGitHub {
     owner = "pazz";
     repo = "alot";
-    rev = version;
-    sha256 = "sha256-1reAq8X9VwaaZDY5UfvcFzHDKd71J88CqJgH3+ANjis=";
+    rev = "refs/tags/${version}";
+    sha256 = "sha256-mXaRzl7260uxio/BQ36BCBxgKhl1r0Rc6PwFZA8qNqc=";
   };
 
   postPatch = ''
     substituteInPlace alot/settings/manager.py \
       --replace /usr/share "$out/share"
```

Comment:
I'd recommend swapping `--replace` for `--replace-fail` while you're in here so we get a heads up if things that are being replaced change upstream.